Engine/Tracker/tracker.py

```diff
@@ -147,6 +159,19 @@ def _change_frame_side(detection_frame: Dict[str, List[Dict[str, Any]]]) -> Dict
 
         return detection_frame
 
+    @staticmethod
+    def _remove_ignored_side(frame):
```

Comment:
I think listing the region that a used is more intuitive than listing the region to ignore. During the competition, I made the mistake often. Instead of having the choice between `None`, `Positive` and `Negative`, you would have the choice between `Full`, `Positive` and `Negative`